crates/services/avsregistry/README.md

```diff
@@ -0,0 +1,3 @@
+# Services AvsRegistry
```

Comment:
add example?

Reply:
we don't have one . leaving it as a placeholder